kobo/apps/organizations/serializers.py

```diff
@@ -1,13 +1,35 @@
 from rest_framework import serializers
 
-from kobo.apps.organizations.models import Organization, create_organization
+from kobo.apps.organizations.models import (
+    create_organization,
+    Organization,
+    OrganizationOwner,
+    OrganizationUser,
+)
+
+
+class OrganizationUserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = OrganizationUser
+        fields = ['user', 'organization']
+
+
+class OrganizationOwnerSerializer(serializers.ModelSerializer):
+    organization_user = OrganizationUserSerializer()
+
+    class Meta:
+        model = OrganizationOwner
+        fields = ['organization_user']
 
 
 class OrganizationSerializer(serializers.ModelSerializer):
+    owner = serializers.CharField(source='owner.organization_user.user.username', read_only=True)
```

Comment:
owner_username? I would expect something called owner to be an object. Did you inspect what sql this generates, I'm surprised it doesn't merit a select/prefetch related but I'm unsure.